stage1/init/common/pod.go

```diff
@@ -348,6 +380,18 @@ func appToSystemd(p *stage1commontypes.Pod, ra *schema.RuntimeApp, interactive b
 		opts = append(opts, unit.NewUnitOption("Service", "SyslogIdentifier", filepath.Base(app.Exec[0])))
 	}
 
+	if flavor == "kvm" {
+		capabilities := kvm.GetAppCapabilities(app)
+		capabilities = append(capabilities, stdCapabilities...)
```

Comment:
If we use the option [systemd-system.conf#CapabilityBoundingSet=](https://www.freedesktop.org/software/systemd/man/systemd-system.conf.html#CapabilityBoundingSet=), it needs to contain the union of stdCapabilities with the union of caps of each app, because systemd-pid1 would only be able to give the caps from its CapabilityBoundingSet, not more.

rkt does that for systemd-nspawn: it adds `--capbility=` for each app, as it iterates over the apps and their caps:
https://github.com/coreos/rkt/blob/master/stage1/init/common/pod.go#L706

But I don't know if we need to keep parity between the nspawn and lkvm flavors?